relay-cardinality/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "relay-cardinality"
+authors = ["Sentry <oss@sentry.io>"]
+description = "Metrics Cardiniality Limiter"
+homepage = "https://getsentry.github.io/relay/"
+repository = "https://github.com/getsentry/relay"
+version = "23.10.1"
+edition = "2021"
+license-file = "../LICENSE"
+publish = false
+
+[features]
+redis = ["relay-redis/impl"]
```

Should there be a `default` feature as well without redis? 

```suggestion
[features]
default = []
redis = ["relay-redis/impl"]
```

---

Isn't that the implicit default behaviour, you always have a default feature which is empty?

---

I meant to have it explicitly.  We have it always defined in our crates. I'm not sure what the team prefer. Explicit or implicit more.

---

Done!